docs/source/reproducers/01-considerations.md

```diff
@@ -0,0 +1,64 @@
+# Environment reproducers general considerations
+## Tested environment
+This reproducer has been successfully tested on a CentOS Stream 9 hypervisor
+running latest CRC qcow2, and latest CentOS Stream 9 qcow2 images.
+
+It *may* work on RHEL based hypervisor and VMs, provided you give the correct
+information for the [discover_latest_image](../roles/discover_latest_image.md)
+role/plugin, or download beforehand the images and consume them from within the
+configuration, but it **wasn't tested** (yet).
+
+## Prerequisites
+### CRC
+The CRC VM has to be properly set up prior running the reproducer play. You
+may get to that leveraging wither the rhol_crc role from the CI Framework, or
```

Comment:
nit: wither -> either

Reply:
i was trying to work out if that was some fancy english there ... :D i mean it almost sounds right 'leverage wither the rhol_crc role'  .. .in my mind it means 'by way of' . k i will stop now.

Reply:
And spellcheck didn't catch it because:
https://www.merriam-webster.com/dictionary/wither

Fun... TIL, as we say ;).